src/themes.js

```diff
@@ -0,0 +1,70 @@
+export const oldTheme = {
+  name: "my-theme",
+  tokens: {
+    fonts: {
+      default: {
+        static: {
+          value: '"Raleway", serif',
+        },
+        variable: {
+          value: '"Raleway", serif',
+        },
+      },
+    },
+    colors: {
+      font: {
+        primary: "#ffffff",
+      },
+      brand: {
+        primary: {
+          10: { value: "rgba(81,120,194,.1)" },
+          20: { value: "rgba(81,120,194,.2)" },
+          40: { value: "rgba(81,120,194,.4)" },
+          60: { value: "rgba(81,120,194,.6)" },
+          80: { value: "rgba(81,120,194,1)" },
+          90: { value: "rgba(81,120,194,.9)" },
+          100: { value: "rgba(81,120,194,1)" },
+        },
+        secondary: {
+          10: { value: "rgba(240,207,96,.1)" },
+          20: { value: "rgba(240,207,96,.2)" },
+          40: { value: "rgba(240,207,96,.4)" },
+          60: { value: "rgba(240,207,96,.6)" },
+          80: { value: "rgba(240,207,96,.8)" },
+          90: { value: "rgba(240,207,96,.9)" },
+          100: { value: "rgba(240,207,96,1)" },
+        },
+      },
+    },
+  },
+};
+
+export const newTheme = {
```

Comment:
commented in `App.js` on this, but styling on login looks off w/new theme.

Reply:
yeah, not implemented yet